hs_core/models.py

```diff
@@ -238,6 +237,66 @@ def page_permissions_page_processor(request, page):
         for edit_grp in edit_groups:
             edit_grp.can_undo = False
 
+    users_json = []
+
+    # TODO: simplify these iterations
+    for usr in owners:
```

Comment:
There are possibilities here, using dictionary/list comprehensions, but they promise to be more clever than readable. What you have here is probably 'least evil', despite the repetition. It is more readable than some alternatives I can imagine.

Reply:
I think it's worth extracting the user json creation into it's own method and then calling the method 5 times while passing in the unique parameters.  It ensures we get the same format for each user type.

Reply:
Do we know how this behaves with non-ascii characters in the user's names?

Reply:
I am not making changes to the way this data is parsed, so it should behave the same.

Reply:
@sblack-usu - I added the method as you suggested and I think it improved readability. Please have another look.